### PR TITLE
Add translated messages in Enums.

### PR DESCRIPTION
**Pull request by chessbr**
Add missing translated messagens in Enums

### CODE REVIEW

File: shoop/core/models/_contacts.py [hunk 1]
```diff
@@ -219,6 +219,11 @@ class Gender(Enum):
     FEMALE = "f"
     OTHER = "o"
 
+    class Labels:
+        UNDISCLOSED = _('undisclosed')
+        MALE = _('male')
+        FEMALE = _('female')
+        OTHER = _('other')
```
Comment (tulimaki) [inline shoop/core/models/_contacts.py]: Missing blank line here.

**Comment by tulimaki** [comment]
@chessbr I think this can be merged. Can you still squash your commits into one and also rebase is needed for the conflicts. Thanks.

**Comment by chessbr** [comment]
Retest this please

**Comment by tulimaki** [comment]
ok to test

**Comment by tulimaki** [comment]
Thanks!

**Pull request merged**